components/content-search/index.js

```diff
@@ -77,7 +90,13 @@ const ContentSearch = ({
 					break;
 			}
 
-			return queryFilter(searchQuery);
+			return queryFilter(searchQuery, {
+				perPage,
+				page,
+				contentTypes,
+				mode,
+				keyword,
+			});
```

Comment:
I realised that you might want all of the arguments used to create a new query there